### PR TITLE
fix(core): skip nx cloud prompt when interactive is false

### PR DESCRIPTION
**Pull request by xiongemi**
## Related Issue(s)

Fixes #

### CODE REVIEW

File: packages/create-nx-workspace/src/internal-utils/prompts.ts [hunk 1]
```diff
@@ -80,6 +84,7 @@ export async function determineDefaultBase(
           message: `Main branch name`,
           initial: `main`,
           type: 'input',
+          skip: !parsedArgs.interactive || isCI(),
```
Comment (FrozenPandaz) [inline packages/create-nx-workspace/src/internal-utils/prompts.ts:87]: Wouldn't it be easier to return without prompting if interactive is false?

**Review by FrozenPandaz — CHANGES_REQUESTED** [review]

File: packages/create-nx-workspace/src/internal-utils/prompts.ts (hunk 1 above)
Comment (xiongemi) [reply packages/create-nx-workspace/src/internal-utils/prompts.ts]: changed

**Review by FrozenPandaz — CHANGES_REQUESTED** [review]

File: packages/create-nx-workspace/src/internal-utils/prompts.ts [hunk 2]
```diff
@@ -70,7 +74,9 @@ export async function determineDefaultBase(
   parsedArgs: yargs.Arguments<{ defaultBase?: string }>
 ): Promise<string> {
   if (parsedArgs.defaultBase) {
-    return Promise.resolve(parsedArgs.defaultBase);
+    return parsedArgs.defaultBase;
+  } else if (!parsedArgs.interactive || isCI()) {
+    return deduceDefaultBase();
```
[the remark this answers is not in the document]
Comment (xiongemi) [reply packages/create-nx-workspace/src/internal-utils/prompts.ts]: changed back

File: packages/nx/src/utils/package-manager.ts [hunk 3]
```diff
@@ -167,6 +169,7 @@ export function getPackageManagerCommand(
           `npm run ${script}${args ? ' -- ' + args : ''}`,
         list: 'npm ls',
         getRegistryUrl: 'npm config get registry',
+        runUninstalledPackage: 'npx --yes',
```
[the remark this answers is not in the document]
Comment (xiongemi) [reply packages/nx/src/utils/package-manager.ts]: changed to use dlx

**Review by FrozenPandaz — APPROVED** [review]

**Pull request merged**